internal/services/appservice/linux_function_app_resource_test.go

```diff
@@ -1211,6 +1211,15 @@ func TestAccLinuxFunctionApp_appStackNodeUpdate(t *testing.T) {
 			),
 		},
 		data.ImportStep(),
+		{
+			Config: r.appStackNode(data, SkuBasicPlan, "22"),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+				check.That(data.ResourceName).Key("kind").HasValue("functionapp,linux"),
+				check.That(data.ResourceName).Key("site_config.0.linux_fx_version").HasValue("NODE|20"),
```

Comment:
Copy paste error here?
```suggestion
			Config: r.appStackNode(data, SkuBasicPlan, "22"),
			Check: acceptance.ComposeTestCheckFunc(
				check.That(data.ResourceName).ExistsInAzure(r),
				check.That(data.ResourceName).Key("kind").HasValue("functionapp,linux"),
				check.That(data.ResourceName).Key("site_config.0.linux_fx_version").HasValue("NODE|22"),
```